pkg/apis/configuration/validation/validation_test.go

```diff
@@ -103,6 +103,13 @@ func TestValidateTLS(t *testing.T) {
 				BasedOn: "scheme",
 			},
 		},
+		{
+			Secret: "my-secret",
```

Comment:
Does it make sense to add cases for the following valid cases as well?
* `Redirect: &v1alpha1.TLSRedirect{}`
*  `Redirect: &v1alpha1.TLSRedirect{Enable: true}`